testrail-connector/src/main/java/com/nullin/testrail/TestRailReporter.java

```diff
@@ -268,6 +270,7 @@ public void reportResult(List<Map<String, Object>> properties) {
 					body.put("comment", new String(comment.toString().getBytes(StandardCharsets.ISO_8859_1),
 							StandardCharsets.UTF_8));
 					body.put("elapsed", elapsed);
+					body.put("custom_test_device",999);
```

Comment:
Same as above: 
Since this is hard-coded field name, let's move it as constant where it's easy to find. Or makes it as another argument with default value.